### ISSUE TITLE
Callout is shifted vertically when tilting the map

### ISSUE THREAD

**Issue by futuretap** [ryanmaxwell/GoogleMapsCalloutView#1]
When the map uses a non-zero viewingAngle, the vertical position of the callout is shifted:

![Bildschirmfoto 2013-04-05 um 14 02 00](https://f.cloud.github.com/assets/110374/343744/a691ec02-9de8-11e2-8cc9-5ff297a53e79.png)

**Comment by ryanmaxwell** [comment]
Yes, I know about this. It's to do with the fact that the callout Y position is a fixed height from the coordinate, but when you're viewing the map at an angled perspective, the height of the pin is relative to the distance from the camera. 

Patches welcome to anybody willing to tackle this!
